src/lib/components/chat/ChatWindow.svelte

```diff
@@ -414,65 +414,67 @@
 					</div>
 				{/if}
 			</div>
-			<form
-				tabindex="-1"
-				aria-label={isFileUploadEnabled ? "file dropzone" : undefined}
-				on:submit|preventDefault={handleSubmit}
-				class="relative flex w-full max-w-4xl flex-1 items-center rounded-xl border bg-gray-100 focus-within:border-gray-300 dark:border-gray-600 dark:bg-gray-700 dark:focus-within:border-gray-500
+			<div class="flex w-full items-center gap-2">
+				<form
```

Comment:
did test extensively but I would prefer not changing the elements wrapping here.

Reply:
yeah not sure why that was there, reverted the changes in that file!